tests/system/action/committee/test_delete.py

```diff
@@ -0,0 +1,71 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class CommitteeDeleteActionTest(BaseActionTestCase):
+    COMMITTEE_ID = 1
+    COMMITTEE_FQID = "committee/1"
+
+    def create_data(self) -> None:
+        self.create_model("organisation/1", {"name": "test_organisation1"})
+        self.create_model("user/20", {"username": "test_user20"})
+        self.create_model("user/21", {"username": "test_user21"})
+
+        self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "committee.create",
+                    "data": [
+                        {
+                            "name": "committee_testname",
+                            "organisation_id": 1,
+                            "description": "<p>Test description</p>",
+                            "member_ids": [20],
+                            "manager_ids": [21],
+                        }
+                    ],
+                }
+            ],
+        )
+
+    def test_delete_correct(self) -> None:
+        self.create_data()
+        response = self.client.post(
+            "/",
+            json=[{"action": "committee.delete", "data": [{"id": self.COMMITTEE_ID}]}],
+        )
+
+        self.assert_status_code(response, 200)
+        self.assert_model_deleted(self.COMMITTEE_FQID)
+        self.assert_model_exists("user/20")
+        self.assert_model_exists("user/21")
```

Comment:
Rather than just checking that the users still exists, you can check that the relations are updated correctly. You can just provide a dict as the second argument which is checked against the model, e.g. in this case `{"member_in_meeting_ids": [], "manager_in_meeting_ids": []}` (I think these are the names of the reverse fields).

Reply:
Done